src/pages/wallet-overview/wallet-overview.html

```diff
@@ -1,46 +1,82 @@
-        </ion-card-header>
-        <ion-card-header>
-          <p id="totalValueText">{{ 'wallet_overview.total_value' | translate }}</p><p id="totalWalletValue">{{ currentWallet.totalCurrentCurrencyValue }}</p><p id="totalWalletValueCurrency">{{ pickedCurrency }}</p>
-        </ion-card-header>
-    </ion-card>
-    <ion-list>
-          <ion-item>
-            <ion-label>{{ 'wallet_overview.choose_value_currency' | translate }}</ion-label>
-            <ion-select [(ngModel)]="pickedCurrency" (ionChange)="setCalculatedCurrencyValue()" interface="action-sheet">
-              <ion-option  *ngFor="let currency of availableCurrencies" [value]="currency">{{ currency }}</ion-option>
-            </ion-select>
-          </ion-item>
-    </ion-list>
-    <ion-card>
-        <ion-card-header>
-            {{ 'wallet_overview.distribution' | translate }}
-            <canvas #doughnutCanvas> </canvas>
-            <div id="doughnutAgenda" class="legend__list" ref="legend">
-                <ul class="legendList" *ngFor="let legendItem of legendList">
-                    <li>
-                        <div class="legendColorBlock" [style.backgroundColor]="legendItem.backgroundColor"></div><p class="legendText">{{ legendItem.label }} {{ legendItem.data }}%</p>
-                    </li>
-                </ul>
-            </div>
-        </ion-card-header>
-    </ion-card>
+    <ion-grid>
+         <ion-row justify-content-center>
+            <ion-col col-12 col-xl-3>
+                <h4 class="wallet-header">{{ 'wallet_overview.wallets' | translate }}</h4>
+                <ion-list radio-group>
+                    <ion-item *ngFor="let wallet of wallets; let i = index;">
+                        <ion-label class="wallet-address">{{ wallet.name }} ({{ wallet.publicKey }})</ion-label>
+                        <ion-radio *ngIf="i === 0" checked="true" (ionSelect)="setCurrentWallet(i)" value="{{ wallet.publicKey }}"></ion-radio>
+                        <ion-radio *ngIf="i > 0" checked="false" (ionSelect)="setCurrentWallet(i)" value="{{ wallet.publicKey }}"></ion-radio>
+                    </ion-item>
+                </ion-list>
+            </ion-col>
+            <ion-col *ngIf="currentWallet" [@visibilityChanged]="walletFundsVisibility" col-xl-3>
+                <h4 class="margin-header">{{ 'wallet_overview.wallet_info' | translate }}</h4>
+                <ion-card>
+                    <ion-card-header *ngFor="let currency of currentWallet.currencies;">
+                        <p class="amount-header">{{ currency.currency }}</p><p class="value-text">{{ currency.amount }}</p>
+                    </ion-card-header>
+                    <ion-card-header>
+                    <p id="total-value-text">{{ 'wallet_overview.total_value' | translate }}</p><p id="total-wallet-value">{{ currentWallet.totalCurrentCurrencyValue }}</p><p id="total-wallet-value-currency">{{ pickedCurrency }}</p>
```

Comment:
Indentation is wrong here. Also the above line is quite long, maybe split it up into several lines?